### PR TITLE
Feat/auth: account sign and refresh, refetch listings when user updated

### PR DESCRIPTION
**Pull request by Xzh0u**
# Description
- account sign and refresh
- refetch listings when user updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "dotenv -e ./.env.development -- react-scripts start",
+    "dev": "dotenv -e ./.env.development -- react-scripts start",
+    "dev:prod": "dotenv -e ./.env.production -- react-scripts start",
```
Comment (rayli09) [inline package.json:7]: update readme on this change as well

File: src/Containers/Nav/Nav.js [hunk 2]
```diff
@@ -56,8 +58,26 @@ export default function ButtonAppBar(props) {
             </Button>
           ) : (
             <GoogleLogin
-              onSuccess={(credentialResponse) => {
-                localStorage.setItem(JWT_NAME, credentialResponse.credential);
+              onSuccess={async (credentialResponse) => {
+                const { credential } = credentialResponse;
+                const resp = await APIs.signAccount(credential);
+
+                if (resp.status !== 200) {
```
Comment (rayli09) [inline src/Containers/Nav/Nav.js:65]: not a blocker, but should we check for specific error status to give more meaningful message?

File: package.json (hunk 1 above)
Comment (Xzh0u) [reply package.json:7]: updated

File: src/Containers/Nav/Nav.js (hunk 2 above)
Comment (Xzh0u) [reply src/Containers/Nav/Nav.js:65]: If the `resp.response.data exists`, it will be shown on the snack bar. I think it's enough

**Review by rayli09 — APPROVED** [review]

**Review by KenXiong123 — APPROVED** [review]

**Pull request merged**